components/post_view/post_attachment_container/post_attachment_container.tsx

```diff
@@ -2,16 +2,42 @@
 // See LICENSE.txt for license information.
 
 import React, {useCallback} from 'react';
+
 import {useHistory} from 'react-router-dom';
 
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+import {isTeamSameWithCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+import {focusPost} from 'components/permalink_view/actions';
+import {GlobalState} from 'types/store';
+
 export type Props = {
     className?: string;
     children?: JSX.Element;
     link: string;
 };
+
+const getTeamAndPostIdFromLink = (link: string) => {
+    const match = (/\/([^/]+)\/pl\/(\w+)/).exec(link);
+
+    return {
+        teamName: match![1],
+        postId: match![2],
+    };
+};
```

Comment:
Sorry for not noticing in my first review :(, 
I think we can change this to use [`matchPath` ](https://reactrouter.com/docs/en/v6/utils/match-path#matchpath) instead of a custom regex

```suggestion
    const {params} = matchPath(url, {path: '/:teamName/pl/:postId'});
    return params;
};
```
(not tested)

Reply:
@koox00 Thanks for the review! 
`matchPath` is very useful😁